img/conference-logos/php-wales-2020.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You added the logo but I don't see the conference on the list

Reply:
Gabby told me to pull it because the list go to long :zipper_mouth_face:

Reply:
It's alright :+1: